dfvfs/vfs/vfs_stat.py

```diff
@@ -41,8 +41,8 @@ def __init__(self):
     self.type = None
 
     # Other stat information.
-    # self.ino = None
-    # self.dev = None
+    self.ino = None
```

Comment:
The VFSStat class is deprecated, use the StatAttribute class instead https://github.com/log2timeline/dfvfs/blob/main/dfvfs/vfs/attribute.py#L24

Reply:
Done.  I've left the changes in VFSStat but can revert if not needed.

Reply:
please revert these

Reply:
FYI pending PR https://github.com/log2timeline/dfvfs/pull/612

Reply:
Done, reverted.